packages/typeorm/src/decorators/typeorm.decorator.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/typeorm
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Context, inject, Injection, ResolutionSession} from '@loopback/core';
+import {QueryRunner} from 'typeorm';
+import {TypeOrmBindings} from '../keys';
+
+export namespace typeorm {
+  export function connection(connectionName?: string) {
+    return inject(
+      '',
+      {decorator: '@typeormConnection'},
```

Comment:
We typically use `typeorm.connection` as the name.

Reply:
Indeed. That's a remnant of something I was trying, will clean it up.